pyproject.toml

```diff
@@ -131,6 +131,7 @@ select = [
     "F401",  # Unused import
     "I",  # All isort rules
     "D101",  # Missing docstring in public class
+    "D102",  # Missing docstring in public method
```

Comment:
```suggestion
```
Removing this line
@pauladkisson this eliminates the check

We still have the problem that I added many #noqa in this PR.

Reply:
Done! Now this is just docstring improvements.